compio/benches/fs.rs

```diff
@@ -64,6 +65,41 @@ fn read_tokio(b: &mut Bencher, (path, offsets): &(&Path, &[u64])) {
     })
 }
 
+fn read_tokio_std(b: &mut Bencher, (path, offsets): &(&Path, &[u64])) {
+    let runtime = tokio::runtime::Builder::new_current_thread()
+        .enable_all()
+        .build()
+        .unwrap();
+    b.to_async(&runtime).iter_custom(|iter| async move {
+        let file = Arc::new(std::fs::File::open(path).unwrap());
+
+        let start = Instant::now();
+        for _i in 0..iter {
+            let mut buffer = [0u8; BUFFER_SIZE];
+            for &offset in *offsets {
+                let file = file.clone();
+                buffer = tokio::task::spawn_blocking(move || {
```

Comment:
In this function, every read operation is performed in a new `spawn_blocking` closure. It should be a little faster than `read_tokio` because it uses `seek_read` or `read_at` instead of `seek` + `read`.